the-content-guide/readme.md

```diff
@@ -1,20 +1,129 @@
+
+Apart from originality, high quality content also makes a certain promise about the credibility of the text. When sources have been used they should provide authority. Good sources are crucial and make sure that a text is taken seriously.
+
+Be a guardian of the originality and authenticity of your content piece. It's easier said than done while we all scrape information from different kinds of sources. When it's authority in people you interview, articles, books or your own judgement, it's all based on other work. The difference where you as an author come in, is how you handle these sources. Combine them to share new insights.
+
+Repetition is not always a bad thing, it can be a powerful tool, but know that it will only work when it's used for original, compelling observations. High quality content is creative and has the potential to excite audiences.
+
+## Structure & readability
+
+Readers are looking for answers and want to find them in the easiest, fastest way. A well structured text gives guidance and leads the reader towards its conclusion in a gentle, organic fashion. A clear text consists of:
+
+* A title
+* An intro
+* Headers
+* **Bold** and *italic*  
+* Short paragraphs
+* Clear quotes
+* A conclusion
+
+A strong and remarkable *title* is important to attract your reader. A good title should activate, interest and excite. Often enough with newspaper articles—also the digital ones—the title is the key component in attracting the reader.
+
+If you have lured your reader with a fantastic title, he/she often decides whether to keep on reading based on the first few sentences. This brings us to the intro of the text and the importance thereof.
+
+> If the title and image are the bait, the intro is the line that has to be strong enough to pull the reader in.
+
+<<<<<<< HEAD
+The *intro* is officiously the first paragraph, and also supports a well chosen title. With this beginning the reader is either triggered and will stay, or not. Readers will scan the intro and decide in just a matter of seconds whether they want to keep on reading. The intro can consist of a promise that you make. Remember that readers tend to find answers to their 'problems'. They were lured in with the title as a hook and now they really want to know if the content is worth there precious time.
+
+**Bold** and *italic* words make sure that highlighted concepts are cleared up. *Short paragraphs* are there to make the text understandable for the reader. *Clear quotes*, then, invite the reader to keep on reading.
+=======
+The *intro* is usually the first paragraph, and also supports a well chosen title. How well an introduction activates the reader determines whether she or he will continue reading the article, this often happens within seconds of seeing the intro for the first time. The intro can consist of a promise that you make, they were lured in with the title as a hook and now they want to know if the content is worth there precious time.
```

Comment:
'How well an introduction activates the reader determines whether she or he will continue reading the article, this often happens within seconds of seeing the intro for the first time.' This is a run-on sentence. Can you see why?

Suggestion: "The introduction's effectiveness in activating the reader..."

'this often happens within seconds of seeing the intro for the first time' is really unclear. Do you mean 'This often happens within seconds of the reader's first encounter with the introduction'?

"The intro can consist of a promise that you make" feels unnecessary.  "Remember that they were lured in with the title as a hook—a promise you made— and now they really want to know if the content is worth their precious time. Showing that it is builds trust."

Also, "there"-->"their" http://www.grammar.cl/english/there-their-they-are.htm